app/serializers/web_hook_user_serializer.rb

```diff
@@ -0,0 +1,33 @@
+class WebHookUserSerializer < UserSerializer
+  def can_edit
+    false
+  end
+
+  def include_email?
```

Comment:
Yes, I made a copy of code from those serializers so that they can be updated and controlled separately for the webhook events.